pkg/core/annotations.go

```diff
@@ -29,12 +29,12 @@ func (c *client) bumpNonceAnnotationForRevision(s *v1alpha1.Service) {
 	if annotations == nil {
 		annotations = map[string]string{}
 	}
-	build := annotations[buildAnnotation]
+	build := annotations[nonceAnnotation]
```

Comment:
Do we want to use more names including `nonce`? I believe @glyn pointed out that this word has a bad connotation in the UK.

Reply:
It's not a big problem. The term has been popular in computer security circles, regardless of the dodgy connotations. Just know that any UK readers are likely to wince, or at least smile wryly, as they read this code.

Reply:
I'm fine with changing this to something other than `nonce`, I think we had `build` originally but changed it, @scothis might have some opinion on this

Reply:
no strong opinion.